src/ocean/util/prometheus/collector/StatFormatter.d

```diff
@@ -0,0 +1,384 @@
+/*******************************************************************************
+
+    Contains methods that format primitive data members from structs or classes
+    into strings that can be exported as responses to prometheus queries.
+
+    Copyright:
+        Copyright (c) 2019 dunnhumby Germany GmbH.
+        All rights reserved
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module ocean.util.prometheus.collector.StatFormatter;
+
+import ocean.net.http.TaskHttpConnectionHandler;
+
+/*******************************************************************************
+
+    Contains methods that format primitive data members from structs or classes
+    into strings that can be exported as responses to prometheus queries.
+
+*******************************************************************************/
+
+public final abstract class StatFormatter
```

Comment:
More idiomatic would be to not use a class and instead declare your functions at module scope. If anyone wants to wrap them in a "namespace", they can do so with a named import.